packages/@okta/vuepress-site/docs/api/getting_started/rate-limits/index.md

```diff
@@ -54,8 +54,10 @@ Note that limits for more specific endpoints override the limits for less specif
 | **Get System Log data:**<br>`/api/v1/events`                                                                 | 20               | 25               | 25      | 50         | 1000               |
 | **Get session information:**<br>`/api/v1/sessions`                                                           | 100              | 300              | 300*    | 600*       | 750                |
 | **Create an organization:**<br>`/api/v1/orgs`                                                                | N/A              | N/A              | N/A     | 50         | 50                 |
-| **Authorize request to a custom Authorization Server:**<br>`/oauth2/{authServerId}/v1/authorize`             | 100              | 300              | 300*    | 600*       | 10000              |
-| **Token request to a custom Authorization Server:**<br>`/oauth2/{authServerId}/v1/token`                     | 100              | 300              | 300*    | 600*       | 10000              |
+| **OAuth2 requests for a Custom Authorization Server:**<br>`/oauth2/{authorizationServerId}/v1`               | 300              | 600              | 600*    | 1200*      | 2000               |
```

Comment:
It is for all custom AS, correct? If yes, then it should be `OAuth2 requests for Custom Authorization Servers`.

Reply:
Good point – will change.